client/src/components/SpotifyContent/SpotifyContent.tsx

```diff
@@ -1,5 +1,154 @@
-function SpotifyContent() {
-  return <p>spoopifycontent--meow</p>;
+import { useEffect } from 'react';
+import { AiFillClockCircle } from 'react-icons/ai';
+import { useAppDispatch } from 'src/store/hooks';
+import { setPlaylistSongs } from 'src/store/musicPlayerSlice';
+import usePlaylist from 'src/utils/usePlaylist';
+import {
+  ContentContainer,
+  ListInfoWrapper,
+  ImageWrapper,
+  DetailWrapper,
+  HeaderRow,
+  HeaderCol,
+  SongListContainer,
+  SongWrapper,
+  SongRow,
+  SongCol,
+  SongColDetail,
+  SongImageWrapper,
+  SongInfoWrapper,
+} from '../../styles/SpotifyContent';
+import { TrackType, ArtistProp, HeaderBkgdType } from '../../types';
+import { convertMsToStandardTime } from '../../utils/Functions';
+
+function SpotifyContent({ headerBackground }: HeaderBkgdType) {
+  const dispatch = useAppDispatch();
+  const playlist = usePlaylist();
+
+  useEffect(() => {
+    const getPlaylistSongs = async () => {
+      const response = await fetch(
+        `/auth/playlists/${playlist.selectedPlaylist}`,
+      );
+      const resData = await response.json();
+      const data = JSON.parse(resData);
+      const songData = {
+        id: data.id,
+        name: data.name,
+        description: data.description.startsWith('<a')
+          ? ''
+          : data.description,
+        image: data.images[0].url,
+        tracks: data.tracks.items.map(({ track }: TrackType) => ({
+          id: track.id,
+          name: track.name,
+          artists: track.artists.map(
+            (artist: ArtistProp) => artist.name,
+          ),
+          image: track.album.images[2].url,
+          duration: track.duration_ms,
+          album: track.album.name,
+          context_uri: track.album.uri,
+          track_number: track.track_number,
+        })),
+      };
+      if (songData) {
+        dispatch(setPlaylistSongs(songData)); // set song data from selected playlist
```

Comment:
since you are using redux, put your api calls into an async thunk

Reply:
Thunking is a good idea. But again, I'm not using axios this project. I've been using axios last few projects, so I'm going back to fetch for this one.